code/modules/mob/living/simple_animal/hostile/retaliate/retaliate.dm

```diff
@@ -21,9 +21,7 @@
 	return see
 
 /mob/living/simple_animal/hostile/retaliate/proc/Retaliate()
-	var/list/around = oview(vision_range, src)
-
-	for(var/atom/movable/A as obj|mob in around)
+	for(var/atom/movable/A as obj|mob in oview(vision_range, src))
```

Comment:
hmmmm

Reply:
blame mechas for forcing multiple loops to be `[o]view` that could've been `[o]viewers` or `[o]hearers`